core/templates/services/UpgradedServices.ts

```diff
@@ -70,6 +70,8 @@ import { BackgroundMaskService } from
   'services/stateful/background-mask.service';
 import { baseInteractionValidationService } from
   'interactions/base-interaction-validation.service';
+import { BottomNavbarStatusService } from
```

Comment:
Could you please add this in `oppia-angular-root.component.ts` as well? Follow point 12 mentioned here: https://github.com/oppia/oppia/wiki/Angular-Migration#implementation-details-to-migrate-services.

Reply:
Done.